src/common/utils.js

```diff
@@ -79,3 +79,16 @@ export class AsyncActionType {
     return `${this.topic}__${this.name}__RESET`;
   }
 }
+
+export function getTodayWithDaysOffset(days) {
+  const date = new Date();
+  date.setUTCDate(date.getUTCDate() - days);
```

Comment:
Why do we need to convert UTCDate back and forth? Why can't we just do `offsetDay = date - days`?